unittests/test_schedulers.py

```diff
@@ -381,11 +392,12 @@ def test_submit(make_job, exec_ctx):
 
     # Additional scheduler-specific checks
     sched_name = minimal_job.scheduler.registered_name
+
     if sched_name == 'local':
         assert [socket.gethostname()] == minimal_job.nodelist
         assert minimal_job.exitcode == 0
         assert minimal_job.state == 'SUCCESS'
-    elif sched_name == ('slurm', 'squeue', 'pbs', 'torque'):
+    elif sched_name == ('slurm', 'squeue', 'pbs', 'torque', 'flux'):
```

Comment:
```suggestion
    elif sched_name in ('slurm', 'squeue', 'pbs', 'torque', 'flux'):
```

Could the `==` be wrong in the first place?

Reply:
Oh, indeed! Good catch!

Reply:
Fixed with https://github.com/reframe-hpc/reframe/pull/2598/commits/e2032ba5884c3380676c976aea90182798e1760c